src/lib/crypto/kimchi_bindings/stubs/rust-toolchain.toml

```diff
@@ -11,4 +11,4 @@
 # 4. figure out the hashes of the (now obsolete) docker images used in CI rules that are failing, grep for these hashes and replace them with the new hashes
 
 [toolchain]
-channel = "1.58.1"
+channel = "1.58.0"
```

Comment:
why? :D

Reply:
also you didn't change the toolchain in the docker files..

Reply:
It was easier than adding the actual toolchain hash in the nix file :D but if this doesn't match docker then I can revert and just add the hash. (We need to bump this to 1.62.0 anyway.)